tests/src/JIT/Regression/JitBlue/GitHub_19222/GitHub_19222.il

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Ensure that the jit doesn't rely on final methods for devirtualization
+// since a final method can be overridden by an explicit .override
+
+.assembly extern mscorlib { auto }
+
+.assembly GitHub_19222 { }
+
+.method assembly static int32 modopt([mscorlib]System.Runtime.CompilerServices.CallConvCdecl) 
+        main() cil managed
+{
+  .entrypoint
+  .maxstack  2
+  .locals ([0] class C c)
+  newobj     instance void C::.ctor()
+  stloc.0
+  newobj     instance void D::.ctor()
+  stloc.0
+  ldloc.0
+  // Must invoke D::f
+  callvirt   instance int32 modopt([mscorlib]System.Runtime.CompilerServices.IsLong) A::f()
```

Comment:
Could you also add a call to C::F that would ensure that the call actually got to C::F and that C::F didn't get overridden to be D::F. This area of the type loader gets very little testing, and this is a decent spot to add one more test.